packages/jest-circus/src/utils.js

```diff
@@ -110,46 +110,59 @@ const getEachHooksForTest = (
   return result;
 };
 
+const _makeTimeoutMessage = (timeout, isHook) => {
+  const message = `Exceeded timeout of ${timeout}ms for a ${isHook ? 'hook' : 'test'}.`;
```

Comment:
What does this look like? Currently, timeouts are the hardest thing to deal with and the signal they give is so bad.

Also, you can write this function without curly braces, return statements or new variables. I believe in you.